addon/ng2/models/webpack-build-common.ts

```diff
@@ -60,10 +60,6 @@ export function getWebpackCommonConfig(projectRoot: string, sourceDir: string) {
     },
     plugins: [
       new ForkCheckerPlugin(),
-      new HtmlWebpackPlugin({
```

Comment:
We have a custom plugin for that which injects `.css` files configured with `autoImported: true` into `index.html`. In this plugin we have also configuration for mobile support which generates content automactically (app-shell).